modules/service-fips.sh

```diff
@@ -47,10 +49,11 @@ fips_is_enabled() {
 fips_check_support() {
     check_service_support \
         "Canonical FIPS 140-2 Modules" "$FIPS_SUPPORTED_SERIES" \
-        "$FIPS_SUPPORTED_ARCHS"
+        "$FIPS_SUPPORTED_ARCHS" || return 1
```

Comment:
check_service_support() just exits (4 or 7), this return 1 is for a "just in case" scenario? Maybe we should change check_service_support() to return 4 or 7 instead if exiting?

Reply:
ah, I think this was a leftover, removed the `|| return 1`